ogre2/src/Ogre2ParticleNoiseListener.hh

```diff
@@ -40,9 +40,19 @@ namespace ignition
       private: virtual void preRenderTargetUpdate(
           const Ogre::RenderTargetEvent &_evt) override;
 
+      /// \brief Pointer to scene
       private: Ogre2ScenePtr scene;
+
+      /// \brief Pointer to camera
       private: Ogre::Camera *ogreCamera = nullptr;
+
+      /// \brief Pointer to ogre matieral with shaders for applying particle
+      /// scattering effect to sensors
       private: Ogre::MaterialPtr ogreMaterial;
+
+      /// \brief Particle scatter ratio. This is used to determine the ratio of
+      /// particles that will detected by sensors
+      private: double particleScatterRatio = 0.65;
```

Comment:
```suggestion
      /// \brief Particle scatter ratio. This is used to determine the ratio of
      /// particles that will detected by sensors. Increasing the ratio increases
      /// the scatter of the particles, making it appear less dense. Decreasing
      /// the ratio decreases the scatter of the particles, making it appear
      /// more dense. This value should be > 0.
      private: float particleScatterRatio = 0.65;
```

I think it'd be good to add a note about how the value of this parameter modifies behavior in case we need to re-visit this later (feel free to modify the documentation as needed if my explanation here is incorrect or unclear).

Also, I think we can change this to be of type `float`, since shaders and the data retrieved from ORGE use floats, right?

Reply:
added and tweaked note. db02aea